sdk/storage/azure-storage-queue/apireview/azure/storage/queue/aio/models.pyi

```diff
@@ -0,0 +1,21 @@
+# Stubs for azure.storage.queue.aio.models (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from .._generated.models import StorageErrorException
+from .._shared.response_handlers import process_storage_error, return_context_and_deserialized
+from ..models import QueueMessage, QueueProperties
+from azure.core.async_paging import AsyncPageIterator
+from typing import Any, Optional
+
+class MessagesPaged(AsyncPageIterator):
+    results_per_page: Any = ...
+    def __init__(self, command: Any, results_per_page: Optional[Any] = ..., continuation_token: Optional[Any] = ...) -> None: ...
+
+class QueuePropertiesPaged(AsyncPageIterator):
+    service_endpoint: Any = ...
```

Comment:
Are these attributes important enough so we want this class to be exposed?
Also, it means there are visible twice today: in azure.storage.queue.aio.models and in azure.storage.queue.aio

Reply:
We should provide a generic protocol for `Paged`.

Reply:
#7863